### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by DominikWolek**

### CODE REVIEW

**Comment by DominikWolek** [comment]
@mat-hek please rename the repository to `membrane_sdk`

File: mix.exs [hunk 1]
```diff
@@ -36,56 +36,60 @@ defmodule Membrane.Gigachad.Mixfile do
...
-      {:membrane_file_plugin, "~> 0.13.2"},
-      {:membrane_common_c, "~> 0.14.0"},
-      {:membrane_mp4_plugin, "~> 0.19.0"},
-      {:membrane_tee_plugin, "~> 0.10.1"},
-      {:membrane_aac_plugin, "~> 0.13.0"},
-      {:membrane_rtp_h264_plugin, "~> 0.15.1"},
-      {:membrane_rtp_opus_plugin, "~> 0.7.0"},
-      {:membrane_rtp_plugin, "~> 0.21.0"},
-      {:membrane_funnel_plugin, "~> 0.7.0"},
-      # {:membrane_ice_plugin, "~> 0.14.3"},
```
Comment (mat-hek) [inline mix.exs:50]: This has a problem with OpenSSL paths, so I think we shouldn't include any Webrtc stuff for now. We can either try to fix it or wait for Jellyfish to finish the work on ex_webrtc, where it should be solved.

File: mix.exs [hunk 2]
```diff
@@ -36,56 +36,60 @@ defmodule Membrane.Gigachad.Mixfile do
...
+      {:membrane_rtp_mpegaudio_plugin, "~> 0.13.0"},
+      {:membrane_audiometer_plugin, "~> 0.11.0"},
+      {:membrane_raw_audio_format, "~> 0.11.0"},
+      {:membrane_camera_capture_plugin, "~> 0.6.0"},
+      {:membrane_ivf_plugin, "~> 0.6.0"},
+      {:membrane_matroska_plugin, "~> 0.4.0"},
+      {:membrane_scissors_plugin, "~> 0.7.0"},
+      {:membrane_flac_plugin, "~> 0.10.0"},
+      {:membrane_ogg_plugin, "~> 0.2.1"},
+      {:membrane_mp3_lame_plugin, "~> 0.16.0"},
```
Comment (mat-hek) [inline mix.exs]: this doesn't have the precompiled library

File: mix.exs [hunk 3]
```diff
@@ -36,56 +36,60 @@ defmodule Membrane.Gigachad.Mixfile do
...
+      {:membrane_rtsp, "~> 0.5.0"},
+      {:ex_sdp, "~> 0.11.0"},
+      {:ex_libsrtp, "~> 0.6.0"},
```
Comment (mat-hek) [inline mix.exs:82]: we should release & use 0.7.0

File: mix.exs [hunk 4]
```diff
@@ -36,56 +36,60 @@ defmodule Membrane.Gigachad.Mixfile do
...
+      {:membrane_video_merger_plugin, "~> 0.8.0"},
+      {:membrane_stream_plugin, "~> 0.3.1"},
+      {:membrane_sdl_plugin, "~> 0.17.0"},
+      {:membrane_ffmpeg_video_filter_plugin, "~> 0.11.0"},
+      {:membrane_ffmpeg_swresample_plugin, "~> 0.18.0"},
+      {:membrane_udp_plugin, "~> 0.10.0"},
+      {:membrane_h264_plugin, "~> 0.7.0"},
+      {:membrane_portaudio_plugin, "~> 0.17.0"},
       {:bunch, "~> 1.6", override: true},
-      {:membrane_mp3_mad_plugin, "~> 0.14.0"},
```
Comment (mat-hek) [inline mix.exs:74]: we should put back the precompiled MAD for Mac ARM

File: mix.exs [hunk 5]
```diff
@@ -36,56 +36,60 @@ defmodule Membrane.Gigachad.Mixfile do
...
+      {:membrane_rtc_engine, "~> 0.17.0"},
+      {:membrane_audio_mix_plugin, "~> 0.15.0"},
+      {:membrane_audio_filler_plugin, "~> 0.2.0"},
+      {:membrane_rtmp_plugin, "~> 0.17.2"},
+      {:membrane_flv_plugin, "~> 0.9.0"},
+      {:membrane_ffmpeg_swscale_plugin, "~> 0.14.0"},
+      {:membrane_video_merger_plugin, "~> 0.8.0"},
+      {:membrane_stream_plugin, "~> 0.3.1"},
+      {:membrane_sdl_plugin, "~> 0.17.0"},
+      {:membrane_ffmpeg_video_filter_plugin, "~> 0.11.0"},
```
Comment (mat-hek) [inline mix.exs]: this has some problems with the precompiled FFmpeg, @Noarkhh is working on that

File: mix.exs [hunk 6]
```diff
@@ -36,56 +36,60 @@ defmodule Membrane.Gigachad.Mixfile do
...
-      {:membrane_mp3_lame_plugin, "~> 0.15.0"},
+      {:membrane_mp3_mad_plugin, "~> 0.17.0"},
+      {:membrane_fake_plugin, "~> 0.10.0"},
+      {:membrane_hackney_plugin, "~> 0.10.0"},
+      {:membrane_raw_video_parser_plugin, "~> 0.11.2"},
+      {:membrane_wav_plugin, "~> 0.9.2"},
+      {:membrane_rtp_mpegaudio_plugin, "~> 0.13.0"},
+      {:membrane_audiometer_plugin, "~> 0.11.0"},
+      {:membrane_raw_audio_format, "~> 0.11.0"},
+      {:membrane_camera_capture_plugin, "~> 0.6.0"},
```
Comment (mat-hek) [inline mix.exs]: this has some problems with the precompiled FFmpeg, @Noarkhh is working on that

File: mix.exs [hunk 7]
```diff
@@ -36,56 +36,60 @@ defmodule Membrane.Gigachad.Mixfile do
...
-      {:membrane_ffmpeg_swresample_plugin, "~> 0.16.1"},
-      {:membrane_udp_plugin, "~> 0.9.2"},
-      {:membrane_h264_plugin, "~> 0.2.1"},
-      {:membrane_portaudio_plugin, "~> 0.15.0"},
+      {:membrane_framerate_converter_plugin, "~> 0.7.0"},
+      {:membrane_generator_plugin, "~> 0.9.0"},
+      {:membrane_rtc_engine, "~> 0.17.0"},
+      {:membrane_audio_mix_plugin, "~> 0.15.0"},
+      {:membrane_audio_filler_plugin, "~> 0.2.0"},
+      {:membrane_rtmp_plugin, "~> 0.17.2"},
```
Comment (mat-hek) [inline mix.exs]: we should add precompiled ffmpeg there

**Review by mat-hek — CHANGES_REQUESTED** [review]
the branch `test` contains deps that work for me on both intel and arm OSX

File: mix.exs (hunk 4 above)
Comment (DominikWolek) [reply mix.exs:74]: https://github.com/membraneframework/membrane_mp3_mad_plugin/pull/45

File: mix.exs (hunk 5 above)
Comment (DominikWolek) [reply mix.exs]: removed

File: mix.exs (hunk 6 above)
Comment (DominikWolek) [reply mix.exs]: removed

File: mix.exs (hunk 7 above)
Comment (DominikWolek) [reply mix.exs]: It caused problems, so it got removed

**Review by mat-hek — APPROVED** [review]

**Pull request merged**